supabase/functions/alerts/alert_types/free_trial_ending.ts

```diff
@@ -16,6 +16,7 @@ type FreeTrialEndingRecord = AlertRecord<"free_trial_ending", FreeTrialEnding>;
 const freeTrialEnding = (req: FreeTrialEndingRecord): EmailConfig[] => {
     return req.arguments.recipients.map((recipient) => ({
         emails: [recipient.email],
+        bcc: !req.arguments.has_credit_card ? ["dave@estuary.dev"] : undefined,
```

Comment:
Dave wants to be included just when a trial is ending without a payment so we can be a bit more proactive.